### ISSUE TITLE
Opening remote file from cli is opening folder

### ISSUE THREAD

**Issue by sandy081** [microsoft/vscode-remote-release#1539]
Testing #1526

`code-insiders --remote ssh-remote+ubuntu16 /home/sandy081/work/vscode/package.json` is opening folder `package.json` 🐛 

![image](https://user-images.githubusercontent.com/10746682/65955172-d3477f80-e447-11e9-85a6-e33d672ff367.png)

**Comment by aeschli** [comment]
Yes, there is guessing involved. Maybe we can say that if it has a extension we treat it as a file. If ending with '/' then it's definitely a folder.
